README.md

```diff
@@ -59,6 +59,40 @@ Please note:
 * Batch sizes specified in prototxt files are per device.
 * Memory layers should not be shared among GPUs, and thus "share_in_parallel: false" is required for layer configuration.
 
+## Building for Spark 2.X
+To Build for Spark 2.X  please include  
+
+* mvn -Dspark2
+
+**Spark2 Pre-release note**  You will need to do the following:
+- in the Spark git clone :
+
+-      mvn **install** \<other spark options\>
+
+Then:  update the CaffeOnSpark caffe-grid/pom.xml in the "spark2" profile section:  set the correct spark (e.g. **2.0.0-SNAPSHOT**),scala (e.g. **2.11.7**)), and hadoop (e.g. **2.7.1**)) versions:
+
+```
+  <profiles>
```

Comment:
Please read :  the updated README itself alread has the answer to your question:

     set the correct spark (e.g. **2.0.0-SNAPSHOT**),scala (e.g. **2.11.7**)), and hadoop (e.g. **2.7.1**)) versions:

The user may need to update the version: after all they may have 2.0.0-preview not SNAPSHOT.